packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -645,29 +656,19 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
     })
 
     describe('Tests Kafka publishing', () => {
-      const mockStream = stream.mockStream(PolicyStreamKafka)
-
-      beforeAll(() => {
-        process.env.KAFKA_HOST = 'localhost:7375'
-      })
-
-      afterAll(() => {
-        delete process.env.KAFKA_HOST
-      })
-      beforeEach(() => {
-        mockStream.write.mockReset()
-      })
-
-      it('Verifies that publishing a policy results in a kafka stream write', async () => {
+      it('Verifies that publishing a policy results in a kafka stream write and includes the publish_date', async () => {
         const policy = PolicyFactory()
 
         await createPolicyAndGeographyFactory(PolicyServiceClient, GeographyServiceClient, policy, {
           publish_date: now()
         })
 
         await PolicyServiceClient.publishPolicy(policy.policy_id, policy.start_date)
+        const publishedPolicy = await PolicyServiceClient.readPolicy(policy.policy_id)
 
         expect(mockStream.write).toHaveBeenCalledTimes(1)
+        const kafkaPolicy = mockStream.write.mock.calls[0][0]
```

Comment:
Could do a double array destructure here if you wanna be super fancy!

```suggestion
        const [[kafkaPolicy]] = mockStream.write.mock.calls
```